projects/angular/clarity.api.md

```diff
@@ -4179,8 +4179,6 @@ export class ClrTabs implements AfterContentInit, OnDestroy {
     // (undocumented)
     resetKeyFocusCurrentToActive(event: FocusEvent): void;
     // (undocumented)
-    get tabIds(): string;

```

Comment:
This is technically breaking, but this property isn't documented.

Reply:
Yes, we have many properties that are only public because they need to be accessed by the template.

Reply:
If it was meaningful outside this context, I'd not remove the property.

Reply:
In v15, we can mark such properties `protected`. But I haven't attempted to do that audit yet.